### DISCUSSION TITLE
4주차 Discussion

### DISCUSSION THREAD

**Discussion by ella-yschoi** [elegant-functional-2023/five-lines-of-code#6] [General]
## 📢 책을 읽으면서 궁금한 점들을 자유롭게 적고 대답해 주세요

4주차 진행 방식

- 읽어오기: 7장 ~ 10장

**Comment by sunhpark42** [comment]
https://eslint.org/docs/latest/rules/consistent-return

**Comment by sunhpark42** [comment]
go, ReScript 에서는 값을 할당하지 않으면 변수를 만들 수 없다. 오호

**Comment by sunhpark42** [comment]
[3. 좋은 함수 만들기 - Null 을 다루는 방법](https://jojoldu.tistory.com/721)
https://jojoldu.tistory.com/721

**Comment by sunhpark42** [comment]
React 18의 useSyncExternalStore, Tearing 현상은 무엇인가?
https://itchallenger.tistory.com/650

**Reply by sunhpark42** [reply]
https://github.com/reactwg/react-18/discussions/69

**Reply by sunhpark42** [reply]
https://tecoble.techcourse.co.kr/post/2021-07-11-suspense/

**Comment by sunhpark42** [comment]
React 패키지의 TODO 주석 - https://github.com/search?q=repo%3Afacebook%2Freact+TODO&type=code

**Comment by sunhpark42** [comment]
워후..npm 의 사건사고
https://namu.wiki/w/npm#s-2.4.1
